pkg/manager/member/template.go

```diff
@@ -160,14 +160,14 @@ then
 echo "waiting for pd cluster ready timeout" >&2
 exit 1
 fi
-
+{{ if eq .CheckDomainScript ""}}
```

Comment:
Will this be a new line if the value is ""?

Reply:
yes.